scripts/notify-checks-result.sh

```
@@ -47,7 +47,8 @@ function get_emoji {
   esac
 }
 run_link="$repo_link/actions/runs/$GITHUB_RUN_ID"
-pylint_formatted="$(get_emoji $PYLINT_RESULT) **Pylint**"
+ruff_formatted="$(get_emoji $RUFF_RESULT) **Ruff**"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)